WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -71,6 +72,13 @@ struct PointOfSaleDashboardView: View {
         .task {
             await posModel.reloadItems(base: .root)
         }
+        .onChange(of: posModel.itemsViewState.containerState) { containerState in
+            if containerState == .loading {
+                withAnimation {
+                    isAnimating = false
+                }
+            }
+        }
```

Comment:
This solves the problem of the floating button going up when the keyboard appears/disappears issue. 

Annoyingly, it has to be applied to the parent, rather than child view but none of our views need to adapt to keyboard safe area. 

This allows to remove `isAnimating` check.

```suggestion
        .ignoresSafeArea(.keyboard)
```